simple_api/django_object/validators.py

```diff
@@ -0,0 +1,29 @@
+from django.core.exceptions import ValidationError
+from simple_api.object.validators import Validator
+
+
+class DjangoValidator(Validator):
+    def __init__(self, field_name, validation_fn):
```

Comment:
`field_name` sem nepatri - pokud validator potrebuje vedet, jake validuje policko, tak to do nej injektuj nejak jinak nez pres konstruktor, ale stejne si myslim, ze to vedet nepotrebuje